influence_benchmark/stats/retroactive_evals.py

```diff
@@ -108,6 +113,29 @@ def load_eval_config(self):
             eval_config[metric]["valid_tokens"] = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
         return eval_config
 
+    def check_and_load_backend(self):
+        if self.using_gpt_backend:
+            if self.backend is None:
+                self.backend = self.backend_class(
+                    model_name=self.backend_config["model_name"],
+                    model_id=self.backend_config["model_id"],
+                    device=None,
+                    max_requests_per_minute=self.backend_config["max_requests_per_minute"],
+                    max_tokens_per_minute=self.backend_config["max_tokens_per_minute"],
+                )
+                print("Loaded backend.")
+        else:
+            if self.backends_dict is None:
+                self.backends_dict = {}
+                for device in self.devices:
+                    self.backends_dict[device] = self.backend_class(
+                        model_name=self.backend_config["model_name"],
+                        model_id=self.backend_config["model_id"],
+                        lora_path=self.backend_config["lora_path"],
+                        device=device,
+                    )
```

Comment:
Also you can just have a single `self.backend_class(device=device, **self.backend_config)` call which should work for both GPT and HF, but still worth keeping the asserts before which are HF or GPT specific